src/build.rs

```diff
@@ -37,41 +38,30 @@ pub fn build(p: &crate::project::Project) -> Result<(), std::io::Error> {
             let metadata = fs::metadata(format!("addons/{}_{}.pbo", p.prefix, name)).unwrap();
             if let Ok(time) = metadata.modified() {
                 if time >= modified {
-                    stdout.set_color(ColorSpec::new().set_fg(Some(Color::White)))?;
-                    println!("  Skipping {}", name);
+                    println!("  {} {}", "Skipping".white().bold(), name);
                     continue;
                 }
+                println!("  {} {}", "Building".green().bold(), name);
+                let mut outf = File::create(&format!("addons/{}_{}.pbo", p.prefix, name))?;
+                armake2::pbo::cmd_build(
+                    path,
+                    &mut outf,
+                    &vec![],
+                    &p.exclude,
+                    &vec![PathBuf::from("./include"), PathBuf::from(".")],
+                )?;
             }
         }
-        stdout.set_color(ColorSpec::new().set_fg(Some(Color::Green)))?;
-        write!(&mut stdout, "  Building ")?;
-        stdout.set_color(ColorSpec::new().set_fg(Some(Color::White)))?;
-        write!(&mut stdout, "{}\r", name)?;
-        stdout.flush()?;
-        let mut outf = File::create(&format!("addons/{}_{}.pbo", p.prefix, name))?;
-        armake2::pbo::cmd_build(
-            path,
-            &mut outf,
-            &vec![],
-            &p.exclude,
-            &vec![PathBuf::from("./include"), PathBuf::from(".")],
-        )?;
-        stdout.set_color(ColorSpec::new().set_fg(Some(Color::Green)))?;
-        write!(&mut stdout, "     Built ")?;
-        stdout.set_color(ColorSpec::new().set_fg(Some(Color::White)))?;
-        writeln!(&mut stdout, "{}", name)?;
     }
     Ok(())
 }
 
-pub fn release(p: &crate::project::Project) -> Result<(), std::io::Error> {
-    let mut stdout = StandardStream::stdout(ColorChoice::Always);
-    let version = match &p.version {
-        Some(v) => v,
-        None => panic!("Unable to determine version number"),
-    };
-    println!("Building Release Version: {}", version);
+pub fn release(p: &crate::project::Project, version: &String) -> Result<(), Error> {
+    println!(" {} release v{}", "Preparing".green().bold(), version);
     build(&p)?;
```

Comment:
Might as well move the build down below the error, otherwise they will need to build it twice.